docs/pages/versions/unversioned/sdk/background-fetch.md

```diff
@@ -11,7 +11,9 @@ import TableOfContentSection from '~/components/plugins/TableOfContentSection';
 
 | Android Device | Android Emulator | iOS Device | iOS Simulator | Web |
 | -------------- | ---------------- | ---------- | ------------- | --- |
-| ✅             | ✅               | ✅         | ✅            | ✅  |
+| ✅ *           | ✅ *             | ✅ *       | ✅ *          | ✅  |
+
+* the feature only works when the app is backgrounded, but not if the app was terminated or upon device reboot.
```

Comment:
```suggestion
### Known issues

BackgroundFetch only works when the app is backgrounded, not if the app has been terminated. [Here is the relevant Github issue](https://github.com/expo/expo/issues/3582)```

Reply:
if the phone reboots, the app is terminated, so I don't think that has to be stated? But I'm not opposed to it if you think that it's better we cover that case specifically

Reply:
Yes, although I'm calling this out specifically because there are Android settings for 'stopOnTerminate' and 'startOnBoot'. Under the covers, Android sends  a signal to all the apps at boot time to ask them if they need to be started so it's slightly different than when the OS is running and the app is terminated.